charts/misc-apps/README.md

```diff
@@ -0,0 +1,53 @@
+# misc-apps
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+Argo CD app-of-apps config for miscellaneous small tools
+
+**Homepage:** <https://github.com/adfinis-sygroup/helm-charts/tree/master/charts/misc-apps>
+
+## Maintainers
+This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk_kwd=helm-charts).
+
+## Source Code
+
+* <https://github.com/adfinis-sygroup/helm-charts>
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://charts.adfinis.com | argoconfig | 0.6.3 |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| downscaler | object | [example](./examples/kube-downscaler.yaml) | [kube-downscaler](https://github.com/hjacobs/kube-downscaler) |
```

Comment:
One (or tow) more points (starting a new thread, since I already resolved the old one):
- donwscaler.name (that is given in values.yaml) is not documented here.
- Also, I do not particularly like that the "example" is linked in the column "Default". Since the example does provide a proposal configuration, and not the used default values, I would consider this to be misleading.

Both of these points do also apply to signalilo (below), and are of course non-functional.

Reply:
fixed in b1bf1f3dfb49a66d84191f4edb5680648ba5dee9

Reply:
This point got it's own issue: #84 

> * Also, I do not particularly like that the "example" is linked in the column "Default". Since the example does provide a proposal configuration, and not the used default values, I would consider this to be misleading.

Reply:
This is definitely better now, and I consider it indeed to be a good idea to open a new issue for this topic, since it should indeed be addressed on a wider basis.